src/schema/ecommerce/approve_order_mutation.js

```diff
@@ -0,0 +1,69 @@
+import { graphql } from "graphql"
+import { OrderReturnType } from "schema/ecommerce/types/order_return"
+import { OrderMutationInputType } from "schema/ecommerce/types/order_mutation_input"
+import { mutationWithClientMutationId } from "graphql-relay"
+
+export const ApproveOrderMutation = mutationWithClientMutationId({
+  name: "ApproveOrder",
+  decription: "Approvess an order with payment",
+  inputFields: OrderMutationInputType.getFields(),
+  outputFields: {
+    result: {
+      type: OrderReturnType,
+      resolve: order => order,
+    },
+  },
+  mutateAndGetPayload: (
+    { orderId },
+    context,
+    { rootValue: { accessToken, exchangeSchema } }
+  ) => {
+    if (!accessToken) {
+      return new Error("You need to be signed in to perform this action")
+    }
+    const mutation = `
+      mutation approveOrder($orderId: ID!) {
+        ecommerce_approveOrder(input: {
+          id: $orderId,
+        }) {
+          order {
+           id
+            code
+            currencyCode
+            state
+            partnerId
+            userId
+            itemsTotalCents
+            shippingTotalCents
+            taxTotalCents
+            commissionFeeCents
+            transactionFeeCents
```

Comment:
lets add `stateUpdatedAt` and `stateExpiresAt` here too, they were added in https://github.com/artsy/metaphysics/pull/1148